CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## Unreleased
+### Fixed
+- Issue with reading a timestamp in a feed #413
+- Passing credentials to a feed #419
+- Background color everywhere #410
+- Updater stopping when one feed fails #408
```

Comment:
I wasn't able to reproduce the "update process stopping" but I can't think of a better wording.

Reply:
I didn't try to reproduce it but I know what would cause it and I fixed that by catching the exceptions.